package.json

```diff
@@ -6,6 +6,7 @@
     "@hotwired/turbo-rails": "^7.2.4",
     "@popperjs/core": "^2.11.6",
     "bootstrap": "^5.2.2",
+    "bulma": "^0.9.4",
```

Comment:
What does this get used for?

Reply:
nothing anymore, it was also added with the coingecko gem for the JS part

Reply:
Can you remove it if it won't be used?